variables.tf

```diff
@@ -164,6 +164,13 @@ variable "node_to_node_encryption_enabled" {
   description = "Whether to enable node-to-node encryption"
 }
 
+variable "encryption_at_rest" {
+  type        = string
```

Comment:
Hi. Thanks for your contribution. Can you please change this to type bool, terraform support bool type.